lib/classes/Error.test.js

```diff
@@ -67,9 +67,15 @@ describe('Error', () => {
   });
 
   describe('#logError()', () => {
+    let BK_SLS_DEBUG;
     beforeEach(() => {
+      BK_SLS_DEBUG = process.env.SLS_DEBUG;
       delete process.env.SLS_DEBUG;
     });
+    afterEach(() => {
+      if (BK_SLS_DEBUG) process.env.SLS_DEBUG = BK_SLS_DEBUG;
+      else delete process.env.SLS_DEBUG;
+    });
```

Comment:
Can we add some `\n` after `BK_SLS_DEBUG` and `beforeEach`?

Out of curiosity: What does `BK` stand for?

Reply:
I believe it translates to _BACKUP_. I actually reused convention that I already found in test files:
https://github.com/serverless/serverless/blob/fdd8335000c003e3138418c2dc7a261bc6e2fc0a/lib/plugins/aws/provider/awsProvider.test.js#L66

If I was to improve it, I would use utility as: https://github.com/medikoo/process-utils#override-env It'll provide cleanest setup for this kind of mangling. Still it won't work in Node.js v4. so I'm relucant to do it before we officially drop that support with: https://github.com/serverless/serverless/pull/6077

(added `\n`)